Directory.Build.targets

```diff
@@ -9,6 +9,7 @@
   </PropertyGroup>
 
   <Import Project="$(RepositoryEngineeringDir)/liveBuilds.targets" />
+  <Import Project="$(RepositoryEngineeringDir)/python.targets" />
```

Comment:
nit: remove the slash as `RepositoryEngineeringDir` already has it in it: https://github.com/dotnet/runtime/blob/master/Directory.Build.props#L24 and https://github.com/dotnet/arcade/blob/e0112fe2f1fd7f6db08b228db98f22225f82267b/src/Microsoft.DotNet.Arcade.Sdk/tools/RepoLayout.props#L44